templates/cloud_init/cloud_init.client.yml

```diff
@@ -0,0 +1,365 @@
+      terraform init
+      [ $? -ne 0 ] && error "terraform init."
+
+      echo "terraform apply scaleup"
+      terraform apply -auto-approve |& tee /tmp/scaleup.txt
+      [ $? -ne 0 ] && error "terraform apply scaleup."
+
+      echo "DONE: end of $0"
+
+  - path: /opt/scaledown.sh
+    permissions: '0755'
+    content: |
+      #!/bin/bash
+
+      function error() {
+        >&2 echo "echo >< ERROR: $* See log for details."
+        exit 1
+      }
+
+      export ARM_SUBSCRIPTION_ID='<sub_id>'
+      export ARM_CLIENT_ID='<arm_client_id>'
+      export ARM_CLIENT_SECRET='<arm_client_scrt>'
+      export ARM_TENANT_ID='<arm_tenant_id>'
+      export TF_VFXT_CONFIG='<terraform_file_config>'
+
+      echo "to file path"
+      cd /tf/src/terraform/examples/vfxt/${TF_VFXT_CONFIG}
+      [ $? -ne 0 ] && error "filepath."
+
+      echo "sed command"
+      sed -i -E -e 's/vfxt_node_count\s*= <new_node_count>/vfxt_node_count = <original_node_count>/' main.tf
+      [ $? -ne 0 ] && error "sed command."
+
+      echo "terraform init"
+      terraform init
+      [ $? -ne 0 ] && error "terraform init."
+
+      echo "terraform apply scaledown"
+      terraform apply -auto-approve |& tee /tmp/scaledown.txt
+      [ $? -ne 0 ] && error "terraform apply scale down."
+
+      echo "DONE: end of $0"
+
+  - path: /opt/destroy.sh
+    permissions: '0755'
+    content: |
+      #!/bin/bash
+
+      function error() {
+        >&2 echo "echo >< ERROR: $* See log for details."
+        exit 1
+      }
+
+      export ARM_SUBSCRIPTION_ID='<sub_id>'
+      export ARM_CLIENT_ID='<arm_client_id>'
+      export ARM_CLIENT_SECRET='<arm_client_scrt>'
+      export ARM_TENANT_ID='<arm_tenant_id>'
+      export TF_VFXT_CONFIG='<terraform_file_config>'
+
+      echo "to file path"
+      cd /tf/src/terraform/examples/vfxt/${TF_VFXT_CONFIG}
+      [ $? -ne 0 ] && error "filepath."
+
+      echo "terraform init"
+      terraform init
+      [ $? -ne 0 ] && error "terraform init."
+
+      echo "terraform destroy"
+      terraform destroy -auto-approve |& tee /tmp/destroy.txt
+      [ $? -ne 0 ] && error "terraform destroy."
+
+      echo "DONE: end of $0"
+
+  - path: /opt/vdbench_destroy.sh
+    permissions: '0755'
+    content: |
+      #!/bin/bash
+
+      function error() {
+        >&2 echo "echo >< ERROR: $* See log for details."
+        exit 1
+      }
+
+      export ARM_SUBSCRIPTION_ID='<sub_id>'
+      export ARM_CLIENT_ID='<arm_client_id>'
+      export ARM_CLIENT_SECRET='<arm_client_scrt>'
+      export ARM_TENANT_ID='<arm_tenant_id>'
+      export REGION='<region>'
+      export RESOURCE_GROUP_NAME='<resource_group_name>'
+      export TF_VFXT_CONFIG='<terraform_file_config>'
+
+      echo "to file path"
+      cd /tf/src/terraform/examples/vfxt/${TF_VFXT_CONFIG}
+      [ $? -ne 0 ] && error "filepath."
+
+      echo "tee outputs to json"
+      terraform output -json |& tee /tmp/tf_output.json
+      [ $? -ne 0 ] && error "tee outputs to json"
```

Comment:
This checks the exit code from `tee`.

Reply:
fixed and updated others to look at pipstatus[0]

Reply:
Looks good! Found one missing instance, but opened a separate comment for that. Resolving this one.